application/src/main/java/org/exoplatform/chat/portlet/notification/NotificationApplication.java

```diff
@@ -76,6 +77,8 @@
 
   SpaceService spaceService_;
 
+  String dbName;
```

Comment:
Is there an instance of this class per request ? We have to be sure there is no leak between tenants.